app/services/importers/crime_map.rb

```diff
@@ -0,0 +1,374 @@
+        'agencies' => [],
+        'layers' => {
+          'selection' => selection
+        },
+        'location' => {
+          'bounds' => {
+            'east' => @east,
+            'north' => @north,
+            'south' => @south,
+            'west' => @west
+          },
+          'lat' => @center_lat,
+          'lng' => @center_lng,
+          'zoom' => @zoom
+        },
+        'analyticLayers' => {
+          'density' => {
+            'selected' => false,
+            'transparency' => 60
+          }
+        }
+      }.to_json
+    end
+    # rubocop:enable Metrics/MethodLength
+
+    # rubocop:disable Metrics/LineLength
+    def pins_headers
+      {
+        accept: 'application/json, text/plain, */*',
+        'accept-language': 'en-US,en;q=0.9,es;q=0.8,he;q=0.7',
+        authorization: "Bearer #{token}",
+        'cache-control': 'no-cache',
+        'content-type': 'application/json',
+        cookie: "ai_user=zPXLlny6Trh2li4D8X3aMv|2024-10-15T21:50:37.649Z; Authorization=Bearer%#{token}; ai_session=dgZ+KN34zuWTPY/g6EAPNV|1729693055912|1729693055912",
+        origin: 'https://communitycrimemap.com',
+        pragma: 'no-cache',
+        priority: 'u=1, i',
+        referer: 'https://communitycrimemap.com/map',
+        'request-id': '|baeb098886174d57ab8a31f3268d4ca3.f4da0b07c76247dc',
+        'sec-ch-ua': '"Google Chrome";v="129", "Not=A?Brand";v="8", "Chromium";v="129"',
+        'sec-ch-ua-mobile': '?0',
+        'sec-ch-ua-platform': '"macOS"',
+        'sec-fetch-dest': 'empty',
+        'sec-fetch-mode': 'cors',
+        'sec-fetch-site': 'same-origin',
+        traceparent: '00-baeb098886174d57ab8a31f3268d4ca3-f4da0b07c76247dc-01',
+        'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36'
+      }
+    end
+    # rubocop:enable Metrics/LineLength
+
+    # rubocop:disable Metrics/LineLength
+    def token
+      return @token if @token
+
+      url = 'https://communitycrimemap.com/api/v1/auth/newToken'
+      headers = {
+        accept: 'application/json, text/plain, */*',
+        'accept-language': 'en-US,en;q=0.9',
+        'cache-control': 'no-cache',
+        pragma: 'no-cache',
+        priority: 'u=1, i',
+        referer: 'https://communitycrimemap.com/map',
+        'sec-ch-ua': '"Google Chrome";v="129", "Not=A?Brand";v="8", "Chromium";v="129"',
+        'sec-ch-ua-mobile': '?0',
+        'sec-ch-ua-platform': '"macOS"',
+        'sec-fetch-dest': 'empty',
+        'sec-fetch-mode': 'cors',
+        'sec-fetch-site': 'same-origin',
+        'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36'
+      }
+      res = HTTParty.get(url, headers: headers)
+      @token = JSON.parse(res.body)['data']['jwt']
+      @token
+    end
+    # rubocop:enable Metrics/LineLength
+
+    # rubocop:disable Metrics/MethodLength
+    def selection
```

Comment:
If I remove this whole thing the request fails..might be able to at least shorten